src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.Fixer.vb

```diff
@@ -19,5 +22,50 @@ Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.Maintainability
                 Return ImmutableArray.Create(BasicRemoveUnusedLocalsAnalyzer.RuleId)
             End Get
         End Property
+
+        Public Sub New()
+            MyBase.New(New BasicNodesProvider())
+        End Sub
+
+        Private Class BasicNodesProvider
+            Inherits NodesProvider
+
+            Public Overrides Function GetNodeToRemoveOrReplace(node As SyntaxNode) As SyntaxNode
+                node = node.Parent
+                If (node.Kind() = SyntaxKind.SimpleAssignmentStatement) Then
+                    Return node
```

Comment:
Any reason the implementation is different from C# one?

Reply:
There are couple reasons: 
1. SyntaxKinds are language specific
2. VB does not support multiple inline assignments: 'a=b=0' has different meaning in VB. Here 'b=0' is a comparison not an assignment.

---
In reply to: [152385002](https://github.com/dotnet/roslyn-analyzers/pull/1336#discussion_r152385002) [](ancestors = 152385002)